### DISCUSSION TITLE
Quota error when try to upload file

### DISCUSSION THREAD

**Discussion by mfertala** [opengisch/QField#3959] [Q&A]
I try to export changes in layer of lines. Message I got is:
`Failed to upload "myLlayer.gpkg". [over_quota] Quota error.`

myLayer is small, 67 features. Where cam be problem with my layer?

**Comment by kcfol** [comment]
Hi @mfertala, 
I had the same issue today when I attempted to sync to the cloud and the issue for me was associated with the licensing option and the size of my project. 
So, if you're using the community option, changing your project to public could allow you to sync the layer to the cloud. 
Or see if your project is greater than 1GB, if it is, deleting some of the versions could help reduce the overall size of your project and help with the sync process. 

I hope that helps.

**Reply by giuenrico93** [reply]
Hi, 
I have a question: how can I delete some of the versions of a project?
Thank you!

**Reply by kcfol** [reply]
Hi, if you login to the QField Cloud online, select your project, select files and then click on the ellipses (three little dots) and from there you'll be able to delete the previous version. 
I hope that  makes sense.

**Reply by cez2020** [reply]
Does the community option (public or private) only allow 1GB-size project uploads? 
I'm trying to upload a project that contains a .mbtiles file that's 5GB heavy and keep getting a quota error. Any advice?

**Reply by giuenrico93** [reply]
> Hi, if you login to the QField Cloud online, select your project, select files and then click on the ellipses (three little dots) and from there you'll be able to delete the previous version. I hope that makes sense.

Thank you!

**Reply by kcfol** [reply]
> Does the community option (public or private) only allow 1GB-size project uploads? I'm trying to upload a project that contains a .mbtiles file that's 5GB heavy and keep getting a quota error. Any advice?

Hey, based on my experience so far, your entire project can't exceed 1 GB regardless of whether you're using the community (public) or pro paid product option (private). The organisation option mentions that you can pay for increased storage but I haven't looked in to the details. 

Are the .mbtile files greater then 1 GB themselves?

**Reply by cez2020** [reply]
> > Does the community option (public or private) only allow 1GB-size project uploads? I'm trying to upload a project that contains a .mbtiles file that's 5GB heavy and keep getting a quota error. Any advice?
> 
> Hey, based on my experience so far, your entire project can't exceed 1 GB regardless of whether you're using the community (public) or pro paid product option (private). The organisation option mentions that you can pay for increased storage but I haven't looked in to the details.
> 
> Are the .mbtile files greater then 1 GB themselves?

The .mbtiles are 5, 6 or 7 GB in size. I've looked into the details of upgrading to a Pro account, and there's a limit of additional 50GB you can purchase, which is a total of EUR 250 + the price of the Pro account. I was counting on the 100 GB the community account came with, but now it's down to only 100 MB (not 1 GB).
